### PR TITLE
Gs resources

### PR DESCRIPTION
**Pull request by DawnWood-Docker**

### CODE REVIEW

**Comment by DawnWood-Docker** [comment]
https://deploy-preview-9682--docsdocker.netlify.com/get-started/resources/

File: _data/toc.yaml [hunk 1]
```diff
@@ -139,8 +141,8 @@ guides:
       path: /get-started/part4/
     - title: "Part 5: Sharing Images on Docker Hub"
       path: /get-started/part5/
-  - path: /engine/docker-overview/
-    title: Docker overview
+  - path: /get-started/resources/
+    title: "Educational Resources"
```
Comment (usha-mandya) [inline _data/toc.yaml]: Should this be Educational resources to be consistent with other TOC entries?

File: get-started/resources.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
+---
+title: "Get Started Educational Resources"
```
Comment (usha-mandya) [inline get-started/resources.md]: Get started educational resources?

Comment (DawnWood-Docker) [reply get-started/resources.md]: Changed to Educational resources to match toc title

File: _data/toc.yaml (hunk 1 above)
Comment (DawnWood-Docker) [reply _data/toc.yaml]: prolly, made the update.

**Comment by usha-mandya** [comment]
Looks good overall. The only observation I had was the capitalization of the title and section headings. Looks good otherwise.

File: get-started/resources.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+\* Courses are fee-based
+
+# Hosted Labs 
+These self-paced and hands-on workshops use free, hosted environments ([Play with Docker](https://labs.play-with-docker.com/) and [Play with Kubernetes](https://labs.play-with-k8s.com/)) that do not require anything to be installed. Follow along and learn more about Docker and Kubernetes.
+
+* [Kubernetes Workshop](https://training.play-with-kubernetes.com/kubernetes-workshop/)
+* [Microservice containerization and orchestration](https://training.play-with-docker.com/microservice-orchestration/)
+
+\* Labs are free but require registration with a Docker ID
```
Comment (traci-morrison) [inline get-started/resources.md]: Are all of the labs free or only the ones marked with an asterisk? I don't see any marked with an asterisk, so just wanted to clarify.

File: get-started/resources.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+* [Bret Fisher](https://www.bretfisher.com/courses/): Docker Mastery, Docker Swarm Mastery, Docker Mastery for Node.js Projects
+* [Nick Janetakis](https://nickjanetakis.com/courses/) Dive into Docker, Docker for DevOps
+* [Nigel Poulton](https://nigelpoulton.com/video-courses): Kubernetes 101, Getting Started with Kubernetes, Docker and Kubernetes: The Big Picture, Kubernetes Deep Dive, Docker Deep Dive
+* [Arun Gupta](https://www.lynda.com/Docker-tutorials/Docker-Java-developers/576584-2.html): Docker for Java Developers
+* [Ajeet Singh Raina](https://collabnix.com/): Docker and Kubernetes Labs
+* [French] [Luc Juggery](https://www.udemy.com/user/lucjuggery/): Introduction to Kubernetes, The Docker Platform
+* [Spanish] [Manuel Morejón](https://www.udemy.com/course/integrando-docker-a-su-infraestrucutra-y-servicios/learn/lecture/6624848#overview): Integrating Docker to its Infrastructure and Services
+
+\* Many of the courses are fee-based
```
Comment (traci-morrison) [inline get-started/resources.md]: Same comment about the asterisk. Is this to distinguish the free courses from the fee-based courses?

File: get-started/resources.md (hunk 3 above)
Comment (DawnWood-Docker) [reply get-started/resources.md]: see other reply

File: get-started/resources.md (hunk 4 above)
Comment (traci-morrison) [reply get-started/resources.md]: Good point. I was not thinking about the way the markdown renders.

**Comment by DawnWood-Docker** [comment]
Hey @traci-morrison can you approve again? :)

**Pull request merged**